components/TransitionHelper/src/CommunityToolkit.Labs.WinUI.TransitionHelper.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="MSBuild.Sdk.Extras/3.0.23">
+  <PropertyGroup>
+    <ToolkitComponentName>TransitionHelper</ToolkitComponentName>
+    <Description>This package contains a TransitionHelper.</Description>
+    <Version>0.0.1</Version>
+
+    <!-- Rns suffix is required for namespaces shared across projects. See https://github.com/CommunityToolkit/Labs-Windows/issues/152 -->
+    <RootNamespace>CommunityToolkit.Labs.WinUI.TransitionHelperRns</RootNamespace>
+
+    <!-- TODO: Putting here as seems to get flagged on different partials, need to deal with TokenSource being disposible -->
+    <!-- See: https://github.com/CommunityToolkit/Labs-Windows/issues/407 -->
+    <NoWarn>CA1001</NoWarn>
```

Comment:
I'm confused as to why this didn't work...
```
  D:\a\Labs-Windows\Labs-Windows\components\TransitionHelper\src\TransitionHelper.Animation.cs(30,29): error CA1001: Type 'TransitionHelper' owns disposable field(s) '_currentAnimationCancellationTokenSource' but is not disposable [D:\a\Labs-Windows\Labs-Windows\components\TransitionHelper\src\CommunityToolkit.Labs.WinUI.TransitionHelper.csproj]
```
@Arlodotexe any thoughts?

I did it here as the CI flags it in the partial `TransitionHelper.Animation.cs` file, but locally it flags it on the `TransitionHelper.Logic.cs` file... 😟

Reply:
Maybe try `<NoWarn>$(NoWarn);CA1001</NoWarn>`?

Reply:
@Arlodotexe just looked, we have a bunch of places in the Tooling props which aren't doing this:
![image](https://user-images.githubusercontent.com/24302614/228311323-86073997-ca70-4518-94e3-2184b16ef0e8.png)

I'll test here if this resolves this issue, but may need a PR on the tooling for this too then.